### DISCUSSION TITLE
[求助]是否可以使用配置文件？

### DISCUSSION THREAD

**Discussion by Rexexe** [aiqinxuancai/discord-proxy#21] [Q&A]
看代码里面有proxy.ini的描述，但是看不懂c++。
现在的情况是，如果手动启动快捷方式，是可以经过代理的。
但是如果开机自启动的话，就没有代理了，用读取配置文件的话应该会好一点

**Comment by aiqinxuancai** [comment]
把快捷方式放到startup文件夹不也能开机启动？
